src/app/query-runner.service.ts

```diff
@@ -161,12 +160,26 @@ export class QueryRunnerService {
     query.duration = (new Date()).getTime() - query.requestSentAt.getTime();
     query.statusCode = res.status;
 
-    AppComponent.messageBarContent = {
+    AppComponent.requestStatus = {
       text: this.createTextSummary(query),
       backgroundClass: this.isSuccessful(query) ? 'ms-MessageBar--success' : 'ms-MessageBar--error',
       icon: this.isSuccessful(query) ? 'ms-Icon--Completed' : 'ms-Icon--ErrorBadge',
     };
 
+    const queries = [
+      'https://graph.microsoft.com/v1.0/me/',
```

Comment:
One thing to think about is whether Canary urls should go through the same treatment as the usual graph urls. So that now you can make this list of queries more generic by using the suffix instead

Reply:
That's a good point. I had not considered canary urls. 

@darrelmiller  @MIchaelMainer  @dkershaw10 

Should we accommodate canary urls?

Reply:
I think more of a concern is making sure this is generic for sovereign clouds, if we have GE in those environments.

Reply:
Let's not forget ppe.